src/js/message.js

```diff
@@ -0,0 +1,131 @@
+import construct from './construct-element';
+import { throwError } from './helpers';
+
+class Message {
```

Comment:
oMessage

Reply:
As far as I can tell, our components that have JavaScript all export the class without the `o`

https://github.com/Financial-Times/o-header/src/js/header.js
https://github.com/Financial-Times/o-tooltip/blob/master/src/js/target.js
https://github.com/Financial-Times/o-banner/blob/master/src/js/banner.js
https://github.com/Financial-Times/o-forms/blob/master/src/js/forms.js

Reply:
Woahhhhhhhhhh. 😲 Right you are. 👍